source/features/suggest-commit-title-limit.tsx

```diff
@@ -21,12 +21,16 @@ function validatePrTitle({delegateTarget: field}: DelegateEvent<Event, HTMLInput
 function init(signal: AbortSignal): void {
 	document.body.classList.add('rgh-suggest-commit-title-limit');
 	onCommitTitleUpdate(validateCommitTitle, signal);
-	delegate('#issue_title', 'input', validatePrTitle, {signal, passive: true});
+	delegate([
+		'#issue_title',
+		'#pull_request_title',
+	], 'input', validatePrTitle, {signal, passive: true});
```

Comment:
Does this need `.join(', ')` ?

```suggestion
	].join(', '), 'input', validatePrTitle, {signal, passive: true});
```

Reply:
Sigh, I thought I had implemented that:

- https://github.com/fregante/delegate-it/issues/35

Anyway for now it's just a type error, all arrays are flattened to `string 1,string 2` before use

Reply:
ah interesting, should I commit the suggestion for now anyway to get rid of the type error?

Reply:
Ok